libraries/postgres/src/Database.js

```diff
@@ -48,26 +48,48 @@ MonitorManager.register({
   },
 });
 
+/** Fixed time comparison of two buffers */
```

Comment:
This seems like a reasonable implementation, but why not just use [crypto.timingSafeEqual(a, b)](https://nodejs.org/api/crypto.html#crypto_crypto_timingsafeequal_a_b)?

Reply:
> There is already [crypto.timingSafeEqual(a, b)](https://nodejs.org/api/crypto.html#crypto_crypto_timingsafeequal_a_b) and you already `require` the `crypto` package, so there is no reason to implement this yourself.

That is a "new" addition (Node 6.6.0) -- this function was copied from azure-entities, and wasn't my handiwork.  We've been using it for years.  Good spot!

> Also note, all changes like these that affect security of the platform should be flagged for security peer review.

Yep, just like the crypto patch, I'm hoping we can get a quick look from Aj based on the fact that this is a quick forward-port of the existing approach.  I pinged on #3082 but I'll send an email now.

Reply:
Can we switch to using `crypto.timingSafeEqual(a, b)` in this PR? If not, can we have a migration plan?

Reply:
Yes, that's part of my followups.  I forgot to work on that yesterday -- sorry!